lib/src/react_client/event_helpers.dart

```diff
@@ -0,0 +1,324 @@
+// ignore_for_file: deprecated_member_use_from_same_package
+
+import '../../react.dart';
+
+/// Returns an empty [SyntheticEvent] with all fields set to `null`.
+///
+/// This utility can be useful when a test just needs access to an event object,
+/// but no fields need to be set.
+SyntheticEvent getEmptySyntheticEvent() => buildSyntheticEvent();
```

Comment:
I don't think these "empty" methods provide enough value to justify having them; for example, why not just call `buildSyntheticEvent" directly?

Reply:
That's definitely fair. I went back and forth myself and my thinking was that from what I saw, this was actually the most common use-case and the intent was to create an empty event. So in application, I felt like this would be the main API and read the best. In contrast `buildSyntheticEvent` to me implied that there's some work to be done like wrapping a pre-existing event or adding specific fields.

I see what you mean though. Like I mentioned, I went back and forth and landed on keeping them so I'd still personally prefer that. Do you feel like they'll ultimately cause more confusion or maintenance? If that's the case I can remove them!

Reply:
I'd prefer to remove them so that we're not providing two ways to do the same thing, and keeping the APIs as simple as possible. Also, if they're using the `createSyntheticEvent` APIs with no arguments, and then later on determine they need to customize a field, all they have to do is add an argument as opposed to having to switch methods.